config.ts

```diff
@@ -0,0 +1,14 @@
+// API Configuration
+
+const API_URL: string = 'https://backend-dex-service-4gxpnuf26a-ez.a.run.app/api/v1/';
+const STAGING_API_URL: string = 'https://backend-dex-service-staging-4gxpnuf26a-ez.a.run.app/api/v1/';
+
```

Comment:
Maybe add these as env variables, not 100% needed as these are not "secrets" per se but good practice regardless.